src/applications/disability-benefits/526EZ/reference-helpers.js

```diff
@@ -1,8 +1,841 @@
+  8957: 'Cyst/Benign Growth - Musculoskeletal - Knee',
+  8958: 'Cyst/Benign Growth - Musculoskeletal - Mid/Lower Back (Thoracolumbar Spine)',
+  8959: 'Cyst/Benign Growth - Musculoskeletal - Neck/Upper Back (Cervical Spine)',
+  8960: 'Cyst/Benign Growth - Musculoskeletal - Other',
+  8961: 'Cyst/Benign Growth - Musculoskeletal - Shoulder',
+  8962: 'Cyst/Benign Growth - Musculoskeletal - Wrist',
+  8963: 'Cyst/Benign Growth - Musculoskeletal and Muscle Injuries',
+  8964: 'Cyst/Benign Growth - Neurological Other System',
+  8965: 'Cyst/Benign Growth - Respiratory',
+  8966: 'Cyst/Benign Growth - Skin',
+  8967: 'Dental and Oral',
+  8968: 'Digestive',
+  8969: 'Ear Disease and Other Sense Organs',
+  8970: 'Eating Disorders',
+  8971: 'Embolism/Infarction - Heart/Veins/Arteries',
+  8972: 'Embolism/Infarction - Neurological Other System',
+  8973: 'Endocrine',
+  8974: 'Eye (Vision)',
+  8975: 'Genitourinary',
+  8976: 'Gulf War Unexplained Chronic Multisymptom Illness',
+  8977: 'Gulf War Undiagnosed Illness',
+  8978: 'Gynecological',
+  8979: 'Heart/Veins/Arteries',
+  8980: 'Helpless Child',
+  8981: 'Hemic and Lymphatic',
+  8982: 'Hemorrhage - Eye (Vision)',
+  8983: 'Hemorrhage - Neurological Other System',
+  8984: 'Herpes - Eye (Vision)',
+  8985: 'Herpes - Skin',
+  8986: 'Infectious Diseases, Immune Disorders and Nutritional Deficiencies',
+  8987: 'Leg Condition - Heart/Veins/Arteries',
+  8988: 'Leg Condition - Neurological Other System',
+  8989: 'Mental Disorders',
+  8990: 'Musculoskeletal - Amputations',
+  8991: 'Musculoskeletal - Ankle',
+  8992: 'Musculoskeletal - Arthritis',
+  8993: 'Musculoskeletal - Elbow',
+  8994: 'Musculoskeletal - Foot',
+  8995: 'Musculoskeletal - Hand',
+  8996: 'Musculoskeletal - Hip',
+  8997: 'Musculoskeletal - Knee',
+  8998: 'Musculoskeletal - Mid/Lower Back (Thoracolumbar Spine)',
+  8999: 'Musculoskeletal - Neck/Upper Back (Cervical Spine)',
+  9000: 'Musculoskeletal - Osteomyelitis',
+  9001: 'Musculoskeletal - Other',
+  9002: 'Musculoskeletal - Shoulder',
+  9003: 'Musculoskeletal - Toe Amputations',
+  9004: 'Musculoskeletal - Wrist',
+  9005: 'Musculoskeletal and Muscle Injuries',
+  9006: 'Neurological - Cranial/Peripheral Nerves',
+  9007: 'Neurological Other System',
+  9008: 'Pension',
+  9009: 'Pension - Aid and Attendance/Housebound',
+  9010: 'Post Traumatic Stress Disorder (PTSD) Combat - Mental Disorders',
+  9011: 'Post Traumatic Stress Disorder (PTSD) Personal Trauma - Mental Disorders',
+  9012: 'Respiratory',
+  9013: 'Scars (Extremities and Trunk)',
+  9014: 'Scars (Head, Face, or Neck)',
+  9015: 'Service-connected Death',
+  9016: 'Skin',
+  9017: 'Special Monthly Compensation - Aid and Attendence/Housebound',
+  9018: 'Special Monthly Compensation - Anatomical Loss/Loss of Use',
+  9019: 'Spina Bifida',
+  9020: 'Traumatic Brain Injury',
+  9021: 'Wound, Bullet/Shell Fragment',
+  249468: 'Cancer - Ear Disease and Other Sense Organs',
```

Comment:
Kind of a big jump from `9021` to `249468` - I'm assuming this list came from somewhere official, but just thought I'd point this out in case it got garbled somehow.

Reply:
Yeah, the list it came from can be found [here](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Disability%20Compensation/discovery/conditions-and-contentions/list-of-800ish-contentions.csv). That list isn't in numerical order, so I sorted them in here for clarity.